edivorce/apps/core/templates/dashboard/final_filing.html

```diff
@@ -135,9 +135,9 @@ <h3>Filing Status</h3>
                         </li>
                         {% if original_marriage_certificate == 'YES' %}
                             <li>{% include "partials/tooltips/forms/proof_of_marriage.html" %}</li>
-                            {% if marriage_certificate_in_english == 'NO' %}
-                                <li>{% include "partials/tooltips/forms/affidavit_of_translation.html" %} sworn / affirmed by a translator</li>
-                            {% endif %}
+                        {% endif %}
+                        {% if marriage_certificate_in_english == 'NO' %}
```

Comment:
@dominikszopa Do you only need this form if the divorce is filed in person? The form is not visible for online filing.
the code looks good.

Reply:
Good point, it should show up in initial e-filing. Updated efiling_documents.py, can you check it is correct?